baxter_demos/BlobInfo[] blobs
